### ISSUE TITLE
Units stuck in unconscious animation when hit by physx object

### ISSUE THREAD

**Issue by diwako** [acemod/ACE3#7403]
**Mods (complete and add to the following information):**
- **Arma 3:** `1.96.146114 Stable`
- **CBA:** `3.13.0` 
- **ACE3:** `3.13.0` 

**Description:**
When a unit is hit by a physx object and was forced to ragdoll, the unit it not able to get up again anymore

**Steps to reproduce:**
- Open eden
- Place 2 units and a quad bike
- preview
- ram the AI with the quad bike

**Expected behavior:**
The unit will be forced to ragdoll, after the recover animation, it will be stuck in the animation `unconscious` and will not get back up anymore. Happens to AI and players.

**Where did the issue occur?**
- Dedicated / Self-Hosted Multiplayer / Singleplayer / Editor (Singleplayer) / Editor (Multiplayer) 

**Additional context:**
Some tries to fix the issue has been tried already. This one will work, however it will run on every unit, and in case the correct condition is met this eventhandler fires multiple times. So it possibly is not an ideal solution.

```sqf
["CAManBase", "AnimDone", {
    params ["_unit", "_anim"];
    if (!alive _unit || {!local _unit || {!(_anim isEqualTo "unconscious") || {_unit getVariable ["ACE_isUnconscious", false] || {lifeState _unit == "INCAPACITATED"}}}}) exitWith {};
    [_unit, false] call ace_medical_engine_fnc_setUnconsciousAnim;
}, true, [], true] call CBA_fnc_addClassEventHandler;
```

**Comment by jonpas** [comment]
Can confirm, happened to me yesterday, getting out of a helicopter and hitting a rearm box someone else was holding just outside, flung 10m away, with all interactions stuck in that animation.

**Comment by PabstMirror** [comment]
Should be fixed by #7499
